app/players/dwarves/dwarf.py

```diff
@@ -0,0 +1,10 @@
+from app.players.player import Player
+
+
+class Dwarf(Player):
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/219366426-13f4bc1e-fc8e-4812-9705-2572f62e4eab.png)
fix this

Reply:
I don't understand exactly what kind of logic I need to put in the Player, Elf and Dwarf classes. Now I have the player_info and get_rating methods in the Player class. I think I don't need to declare them in the Elf and Gnome classes, because I can inherit them from the Player class. Can you help me figure it out?